docs/business-apps/introduction-to-sharepoint-business-process-integration.md

```diff
@@ -1,46 +1,46 @@
 ---
-title: SharePoint custom forms and automated business workflow
-description: SharePoint custom forms and automated business workflow.
+title: Business apps and business process automation
+description: Business apps and business process automation.
 ms.date: 02/02/2018
```

Comment:
Please update the `ms.date` to the current date to reflect when the updated doc is published.